docs/getting-started/overview.md

```diff
@@ -0,0 +1,62 @@
+# Getting Started
+
+Welcome to STAC Server! This guide will help you understand what STAC Server is and how to get started.
+
+## What is STAC?
+
+The **[SpatioTemporal Asset Catalog (STAC)](https://stacspec.org/)** specification provides a common language to describe geospatial information, making it more easily discoverable and accessible. STAC enables interoperability between systems and allows users to search across multiple catalogs.
+
+## What is STAC Server?
+
+STAC Server is a production-ready implementation of the [STAC API specification](https://github.com/radiantearth/stac-api-spec). It provides:
+
+- **RESTful API**: Search and access spatiotemporal data via HTTP
+- **Elasticsearch Backend**: Scalable indexing and search capabilities
```

Comment:
OpenSearch. I'm not certain but changelog for v2.0.0 states "Elasticsearch is no longer supported as a backend. Only OpenSearch is now supported."